sklearn/inspection/tests/test_partial_dependence.py

```diff
@@ -583,7 +583,7 @@ def test_partial_dependence_pipeline():
 
 @pytest.mark.parametrize(
     "estimator",
-    [LogisticRegression(max_iter=1000, random_state=0),
+    [LogisticRegression(random_state=0, C=1e-3),
```

Comment:
Could we scale the data as well (with a pipeline or the full dataset).

Reply:
I updated the PR with scaling the data.